src/Symfony/Component/Intl/Resources/emoji/composer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "repositories": [
+        {
+            "type": "package",
+            "package": {
+                "name": "unicode-org/cldr",
+                "version": "2022.06.29",
+                "source": {
+                    "type": "git",
+                    "url": "https://github.com/unicode-org/cldr",
+                    "reference": "production/2022-06-29-1740z"
```

Comment:
not without having a composer.json in the repository or using a manual management of the install instead of using composer. The `type=package` does not work well with a mutable target

Reply:
got it, then let's add a note about this in the readme